packages/ui/src/stories/Button/Button.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
```

Comment:
This file is now `Button/Button.tsx` can we make it `Button/index.tsx`? 

I think you will see the reasons, but if not we can discuss it

Reply:
Already done. I see the reasons. I mean, I see how it helps us with the imports, maybe there are more reasons I don't know about.

Reply:
Exactly that!